dist/steps/build.sh

```diff
@@ -101,20 +100,14 @@ echo "#    Project directory    #"
 echo "###########################"
 echo ""
 
-ls -alh "$UNITY_PROJECT_PATH"
-
-#
-# Build
-#
+ls -alh $UNITY_PROJECT_PATH
 
 echo ""
 echo "###########################"
-echo "#    Building project     #"
+echo "#    Building platform    #"
```

Comment:
Aren't we building a project, just like in other use cases?

Reply:
Sorry, these appear to be a merging issues (i.e not changes I made). Will fix and re-open.

Reply:
No problem of course :)